int8 robot
string PoseName
bool isJointSpace
---
bool isFinish